Lombiq.Privacy.Tests.UI/Extensions/TestCaseUITestContextExtensions.cs

```diff
@@ -0,0 +1,120 @@
+    }
+
+    public static async Task TestFormConsentCheckboxContentAsync(this UITestContext context)
+    {
+        await context.GoToRelativeUrlAsync(AbsolutePaths.CompetitorRegistration);
+
+        context.VerifyElementTexts(
+            By.CssSelector(ElementSelectors.PrivacyConsentCheckboxLabel),
+            new[] { ExpectedContents.FormConsentCheckboxContent });
+    }
+
+    public static async Task TestConsentBannerAsync(this UITestContext context)
+    {
+        await context.EnablePrivacyConsentBannerFeatureAsync();
+        await context.TestConsentBannerContentAsync();
+        await context.TestConsentBannerAcceptButtonAsync();
+    }
+
+    public static async Task TestConsentBannerContentAsync(this UITestContext context)
+    {
+        await context.GoToHomePageAsync();
+
+        context.VerifyElementTexts(
+            By.CssSelector("#privacy-consent-banner > .modal-content > .modal-body"),
+            new[] { ExpectedContents.ConsentBannerContent });
+    }
+
+    public static async Task TestConsentBannerAcceptButtonAsync(this UITestContext context)
+    {
+        var privacyConsentAcceptButton = By.Id(ElementSelectors.PrivacyConsentAcceptButtonId);
+
+        await context.GoToHomePageAsync();
+
+        await context.ClickReliablyOnAsync(privacyConsentAcceptButton);
+        context.Missing(privacyConsentAcceptButton);
+
+        // Verify persistence.
+        context.Refresh();
+        context.Missing(privacyConsentAcceptButton);
+    }
+
+    public static async Task TestRegistrationConsentCheckboxAsync(this UITestContext context)
+    {
+        await context.EnablePrivacyConsentBannerFeatureAsync();
+        await context.EnablePrivacyRegistrationConsentFeatureAsync();
+        await context.SetUserRegistrationTypeAsync(UserRegistrationType.AllowRegistration);
+
+        await context.TestRegistrationConsentCheckboxContentAsync();
+
+        // Go to registration and create a new user
+        await context.GoToRegistrationPageAsync();
+        await context.FillInWithRetriesAsync(By.Id("UserName"), TestUser.Name);
+        await context.FillInWithRetriesAsync(By.Id("Email"), TestUser.Email);
+        await context.FillInWithRetriesAsync(By.Id("Password"), TestUser.Password);
+        await context.FillInWithRetriesAsync(By.Id("ConfirmPassword"), TestUser.Password);
+        await context.SetCheckboxValueAsync(By.Id("RegistrationCheckbox"), isChecked: true);
+        await context.ClickReliablyOnSubmitAsync();
+
+        // Login with the created user
+        await context.GoToRelativeUrlAsync("/Login");
+        await context.FillInWithRetriesAsync(By.Id("UserName"), TestUser.Name);
+        await context.FillInWithRetriesAsync(By.Id("Password"), TestUser.Password);
+        await context.ClickReliablyOnSubmitAsync();
+        (await context.GetCurrentUserNameAsync()).ShouldBe(TestUser.Name);
+
+        // Check that, the consent banner not comes up after login
```

Comment:
```suggestion
        // Check that, the consent banner doesn't come up after login.
```
Please add period to the other comments in this file as well.